src/iop/demosaic.c

```diff
@@ -101,6 +102,8 @@ typedef struct dt_iop_demosaic_global_data_t
   int kernel_zoom_third_size;
   int kernel_vng_green_equilibrate;
   int kernel_vng_interpolate;
+
+  double CAM_to_RGB[3][4];
```

Comment:
Thanks! that actually fixes an intermittent bug I was seeing when switching images.